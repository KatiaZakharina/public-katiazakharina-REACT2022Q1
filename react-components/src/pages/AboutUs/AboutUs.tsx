import { Hero } from 'components/Hero/Hero';
import { InfoSection } from './InfoSection/InfoSection';
import { ColumnContainer } from 'components/Layout/Container';
import { Main } from 'components/Layout/Main';

import about from 'assets/bg_img/about.jpg';
import about1 from 'assets/bg_img/about1.webp';
import about2 from 'assets/bg_img/about2.webp';
import { Side } from './InfoSection/SideEnum';

export function AboutUs() {
  return (
    <Main data-testid="about_page">
      <Hero bg={about} text={'The relentless pursuit of excellence'} />
      <ColumnContainer>
        <InfoSection side={Side['right']} img={about1}>
          <h2>A transformative adventure</h2>
          <p>
            Lorem ipsum dolor sit amet consectetur adipiscing elit ut aliquam purus sit amet luctus
            venenatis lectus magna fringilla urna porttitor rhoncus dolor purus non enim praesent
            elementum facilisis leo vel fringilla est ullamcorper eget nulla facilisi etiam
            dignissim diam.
          </p>
          <p>
            Quis enim lobortis scelerisque fermentum dui faucibus in ornare quam viverra orci
            sagittis eu volutpat odio facilisis mauris sit amet massa vitae tortor condimentum
            lacinia quis vel eros donec ac odio tempor orci dapibus ultrices in iaculis nunc sed
            augue lacus
          </p>
        </InfoSection>
        <InfoSection side={Side['left']} img={about2}>
          <h2>Somewhere, something, someone â€” new</h2>
          <p>
            Lorem ipsum dolor sit amet consectetur adipiscing elit ut aliquam purus sit amet luctus
            venenatis lectus magna fringilla urna porttitor rhoncus dolor purus non enim praesent
            elementum facilisis leo vel fringilla est ullamcorper eget nulla facilisi etiam
            dignissim diam.
          </p>
          <p>
            Quis enim lobortis scelerisque fermentum dui faucibus in ornare quam viverra orci
            sagittis eu volutpat odio facilisis mauris sit amet massa vitae tortor condimentum
            lacinia quis vel eros donec ac odio tempor orci dapibus ultrices in iaculis nunc sed
            augue lacus
          </p>
        </InfoSection>
      </ColumnContainer>
    </Main>
  );
}
