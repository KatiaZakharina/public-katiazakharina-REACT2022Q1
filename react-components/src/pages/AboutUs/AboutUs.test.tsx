import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import { AboutUs } from './AboutUs';

describe('AboutUs', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<AboutUs />);
    expect(asFragment()).toMatchSnapshot();
  });
});
