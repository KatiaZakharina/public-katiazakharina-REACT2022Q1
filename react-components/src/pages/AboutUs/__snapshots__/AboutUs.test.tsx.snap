// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs renders correctly 1`] = `
<DocumentFragment>
  <main
    class="sc-jRQBWg dFLTbQ"
    data-testid="about_page"
  >
    <div
      class="sc-dkPtRN lnasxi"
      data-testid="hero_section"
    >
      <section
        class="sc-bdvvtL kcgmgL"
      >
        <h2
          class="sc-hKwDye MrYYA"
        >
          The relentless pursuit of excellence
        </h2>
      </section>
    </div>
    <section
      class="sc-bdvvtL sc-gsDKAQ kcgmgL kkVWFs"
    >
      <div
        class="sc-eCImPb bzSFHD"
      >
        <div>
          <h2>
            A transformative adventure
          </h2>
          <p>
            Lorem ipsum dolor sit amet consectetur adipiscing elit ut aliquam purus sit amet luctus venenatis lectus magna fringilla urna porttitor rhoncus dolor purus non enim praesent elementum facilisis leo vel fringilla est ullamcorper eget nulla facilisi etiam dignissim diam.
          </p>
          <p>
            Quis enim lobortis scelerisque fermentum dui faucibus in ornare quam viverra orci sagittis eu volutpat odio facilisis mauris sit amet massa vitae tortor condimentum lacinia quis vel eros donec ac odio tempor orci dapibus ultrices in iaculis nunc sed augue lacus
          </p>
        </div>
        <img
          alt="mountain landscape"
          src="about1.webp"
        />
      </div>
      <div
        class="sc-eCImPb igEKha"
      >
        <div>
          <h2>
            Somewhere, something, someone â€” new
          </h2>
          <p>
            Lorem ipsum dolor sit amet consectetur adipiscing elit ut aliquam purus sit amet luctus venenatis lectus magna fringilla urna porttitor rhoncus dolor purus non enim praesent elementum facilisis leo vel fringilla est ullamcorper eget nulla facilisi etiam dignissim diam.
          </p>
          <p>
            Quis enim lobortis scelerisque fermentum dui faucibus in ornare quam viverra orci sagittis eu volutpat odio facilisis mauris sit amet massa vitae tortor condimentum lacinia quis vel eros donec ac odio tempor orci dapibus ultrices in iaculis nunc sed augue lacus
          </p>
        </div>
        <img
          alt="mountain landscape"
          src="about2.webp"
        />
      </div>
    </section>
  </main>
</DocumentFragment>
`;
